pkg/kubectl/cmd/rollout/rollout_history.go

```diff
@@ -88,9 +88,7 @@ func RunHistory(f *cmdutil.Factory, cmd *cobra.Command, out io.Writer, args []st
 		Flatten().
 		Do()
 	err = r.Err()
-	if err != nil {
-		return err
-	}
+	cmdutil.CheckErr(err)
```

Comment:
Instead of calling `CheckErr` here, you should do it in the upper level. Right now we are aggregating all those errors but each step should fail to run if an error is encountered.